### PR TITLE
further clarify background scans

### PR DESCRIPTION
**Pull request by JimBugwadia**

### CODE REVIEW

File: content/en/docs/Writing policies/background.md [hunk 1]
```diff
@@ -14,14 +20,12 @@ spec:
...
 
-By default, background scanning occurs every 15 minutes and is not currently configurable.
+When background scans are enabled for a policy, if existing resources are found which would violate the policy, the background scan reports them using a `ClusterPolicyReport` or a `PolicyReport` object, depending on if the resource is namespaced or not. 
 
-{{% alert title="Note" color="info" %}}
-⚠️  Kyverno does not mutate existing resources to prevent inadvertent changes to workloads.
-Mutate and generate rules are not processed during background scans.
-{{% /alert %}}
+The bachround scan does not block, or otherwise impact, any existing resources that match a `validate` rule, even in `enforce` mode. Background scanning is an optional field and defaults to `true`, only taking effect on `validate` rules. It has no effect on either `generate` or `mutate` rules.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/background.md:26]: "The background scan..."

File: content/en/docs/Writing policies/background.md [hunk 2]
```diff
@@ -14,14 +20,12 @@ spec:
...
+
+By default, background scanning occurs at start and then every 15 minutes. Currently the scan interval is not configurable.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/background.md:28]: "occur at start" is confusing to me. Start of what? Start of creating a policy? Either clarify what that "start" trigger is or omit.

File: content/en/docs/Writing policies/background.md [hunk 3]
```diff
@@ -39,4 +43,5 @@ When background scanning is enabled, regardless of whether the policy's `validat
 | `validationFailureAction: enforce` | None         | None              |
 | `validationFailureAction: audit`   | Report       | None              |
 
-Also, policy rules that are written using variables from [Admission Review](/docs/writing-policies/variables/#variables-from-admission-review-request-data) request information (e.g. `{{request.userInfo}}`) cannot be applied to existing resources in the background scanning mode since that information is not available. Hence, these rules must set `background` to `false` to disable background scanning.
+
+Policy rules that are written using variables from [Admission Review](/docs/writing-policies/variables/#variables-from-admission-review-request-data) request information (e.g. `{{request.userInfo}}`) cannot be applied to existing resources in the background scanning mode since that information is not available. Hence, these rules must set `background` to `false` to disable background scanning.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/background.md:47]: As an aside, is this check being included in future policy schema validations?

**Review by chipzoller — COMMENTED** [review]
Changes look good overall, just a couple of points and a misspelling.

File: content/en/docs/Writing policies/background.md (hunk 3 above)
Comment (JimBugwadia) [reply content/en/docs/Writing policies/background.md:47]: yes, it already is....

File: content/en/docs/Writing policies/background.md (hunk 2 above)
Comment (JimBugwadia) [reply content/en/docs/Writing policies/background.md]: Its when Kyverno starts. Clarified!

File: content/en/docs/Writing policies/background.md (hunk 1 above)
Comment (JimBugwadia) [reply content/en/docs/Writing policies/background.md]: fixed, thanks!

**Review by chipzoller — APPROVED** [review]

**Pull request merged**